pkg/generator/warewulf.go

```diff
@@ -51,47 +44,30 @@ func (g *Warewulf) Generate(config *configurator.Config, opts ...util.Option) (F
 		return nil, fmt.Errorf("no ethernet interfaces found")
 	}
 
-	// print message if verbose param found
-	if verbose, ok := params["verbose"].(bool); ok {
-		if verbose {
-			fmt.Printf("template: \n%s\n ethernet interfaces found: %v\n", strings.Join(target.TemplatePaths, "\n\t"), len(eths))
-		}
-	}
-
 	// fetch redfish endpoints and handle errors
-	eps, err := client.FetchRedfishEndpoints(opts...)
+	eps, err := smdClient.FetchRedfishEndpoints(params.Verbose)
 	if err != nil {
 		return nil, fmt.Errorf("failed to fetch redfish endpoints: %v", err)
 	}
 	if len(eps) <= 0 {
 		return nil, fmt.Errorf("no redfish endpoints found")
 	}
 
-	// format output for template substitution
-	nodeEntries := ""
-
-	// load files and templates and copy to outputs
-	files, err := LoadFiles(target.FilePaths...)
-	if err != nil {
-		return nil, fmt.Errorf("failed to load files: %v", err)
-	}
-	templates, err := ApplyTemplateFromFiles(Mappings{
+	templates, err := ApplyTemplates(Mappings{
 		"node_entries": nodeEntries,
-	}, target.TemplatePaths...)
+	}, params.Templates)
 	if err != nil {
 		return nil, fmt.Errorf("failed to load templates: %v", err)
 	}
 
-	maps.Copy(outputs, files)
+	maps.Copy(outputs, params.Files)
 	maps.Copy(outputs, templates)
 
 	// print message if verbose param is found
-	if verbose, ok := params["verbose"].(bool); ok {
-		if verbose {
-			fmt.Printf("templates and files loaded: \n")
-			for path, _ := range outputs {
-				fmt.Printf("\t%s", path)
-			}
+	if params.Verbose {
+		fmt.Printf("templates and files loaded: \n")
```

Comment:
Replace with `fmt.Println("templates and files loaded:")`?

Reply:
I'm going to change this to use `log.Info()` instead.